source/devices/AM62AX/linux/Release_Specific_Release_Notes.rst

```diff
@@ -45,46 +45,41 @@ What's new
 
 **Processor SDK Linux AM62A Release has following new features:**
 
-  - LTS Refresh with Stable Kernel update to 6.6.32
-  - Important Bug Fixes on top of Processor SDK 09.02.01.10 Release
-  - Kernel: MMC Highspeed tuning and Performance Improvements
-  - Mini Fusion board support
-  - LI OX05B1S Sensor support
-  - `CC33XX M.2 Module <https://www.ti.com/tool/M2-CC3351>`__ Wi-Fi support
-  - Kernel: MMC Highspeed tuning and Performance Improvements
-  - RT Kernel : Significant Improvements to Real-Time Linux Interrupt Latency, :ref:`RT Interrupt Latencies <RT-linux-performance>`
-  - TIFS, DM, R5 SPL: PLL Configuration sequence updated to latest recommendation
-  - TISCI ABI version updated to 4.0 from 3.1 (Low power mode related API changes)
+  - LTS Stable Kernel update to 6.6.58
+  - Important Bug Fixes on top of Processor SDK 10.00.07.04 Release
+  - RT Kernel : Real-Time Linux Interrupt Latency numbers here :ref:`RT Interrupt Latencies <RT-linux-performance>`
 
 **Component version:**
 
-  - Kernel 6.6.32
-  - RT Kernel 6.6.32-rt32
+  - Kernel 6.6.58
+  - RT Kernel 6.6.58-rt45
   - U-Boot 2024.04
   - Toolchain GCC 13.3
-  - ATF 2.10+
-  - OPTEE 4.2.0
-  - TIFS Firmware v10.00.08
-  - DM Firmware 10.00.00.08
+  - ATF 2.11+
+  - OPTEE 4.4.0
+  - Graphics DDK 24.1
```

Comment:
@praneethbajjuri is this applicable for AM62A which doesn’t has a GPU?

Reply:
good catch , copy paste error